lib/iris/cube.py

```diff
@@ -3161,15 +3161,35 @@ def __hash__(self):
 
     def __add__(self, other):
         return iris.analysis.maths.add(self, other, ignore=True)
+
+    def __iadd__(self, other):
+        return iris.analysis.maths.add(self, other, ignore=True, in_place=True)
+
     __radd__ = __add__
 
     def __sub__(self, other):
         return iris.analysis.maths.subtract(self, other, ignore=True)
 
+    def __isub__(self, other):
+        return iris.analysis.maths.subtract(self, other,
+                                            ignore=True, in_place=True)
+
     __mul__ = iris.analysis.maths.multiply
     __rmul__ = iris.analysis.maths.multiply
+
+    def __imul__(self, other):
+        return iris.analysis.maths.multiply(self, other, in_place=True)
+
     __div__ = iris.analysis.maths.divide
```

Comment:
Hang on, this might be a problem... I've asserted here that we should _only_ use `truediv` when dividing cubes, yet as it stands if I do `my_cube / 5`, it is _this_ codepath that will be followed and not `truediv`. This can't be right.